### PR TITLE
feat(theme/colors): fix colors in accordance with Frontify

### PR DESCRIPTION
**Pull request by mtikhinskaya**
**What is the problem this PR is trying to solve?**
colors do not satisfy Frontify description

**What is the chosen solution to this problem?**
colors satisfy Frontify description
http://guidelines.talend.com/document/92132#/design-principles/colors-in-app

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/theme/src/theme/_variables.scss [hunk 1]
```diff
@@ -17,13 +17,13 @@ $gray:                   lighten($gray-base, 33.5%) !default; // #555
 $gray-light:             lighten($gray-base, 46.7%) !default; // #777
 $gray-lighter:           lighten($gray-base, 85%) !default;   // #bfbfbf
 
-$brand-primary:         $st-tropaz !default;
+$brand-primary:         $patina !default;
```
Comment (talend-sromain) [inline packages/theme/src/theme/_variables.scss]: Why change this one ?

Comment (mmaillart) [inline packages/theme/src/theme/_variables.scss]: 1. Shouldn't we remove the constant **$limeade** (the previous SUCCESS_GREEN) to make sure it is not used anymore?
2. I am wondering where the **$green-talend** is used ?

File: packages/theme/src/theme/_colors.scss [hunk 2]
```diff
@@ -3,93 +3,129 @@
...
 /// @type Color
-$rio-grande: #C3D600;
+$st-tropaz: #236192;
 
-/// Contextual color for informational alert messages
+/// Green Talend
 ///
 /// @type Color
-$scooter: #66BDFF;
+$green-talend: #B6BE00;
```
Comment (mmaillart) [inline packages/theme/src/theme/_colors.scss]: I am wondering where the $green-talend is used ?

File: packages/theme/src/theme/_colors.scss [hunk 3]
```diff
@@ -3,93 +3,129 @@
...
+
+////
+/// Deprecated colors
+/// These colors are deprecated
+/// @group DeprecatedColors
+////
+
+/// Should be changed to $rio-grande
+/// @type Color
+$limeade: #8ABC00;
```
Comment (mmaillart) [inline packages/theme/src/theme/_colors.scss]: Shouldn't we remove the constant $limeade (the previous SUCCESS_GREEN) to make sure it is not used anymore?

File: packages/theme/src/theme/_colors.scss (hunk 2 above)
Comment (jmfrancois) [inline packages/theme/src/theme/_colors.scss]: it's define in the guidelines so it's must be available

**Comment by jsomsanith** [comment]
Close this one for #629

**Pull request closed without merging**